src/patternfly/components/Timestamp/timestamp.scss

```diff
@@ -2,13 +2,14 @@
 
 :where(:root, .#{$timestamp}) {
```

Comment:
Super nit, would you mind adding a new timestamp to the existing list (or whatever you think is best) that shows the help text styling? Doesn't look like we have an example of that anywhere in the core docs.

You can get it by adding `timestamp--IsHelp=true` to the outer `{{#> timestamp}}` so it's `{{#> timestamp timestamp--IsHelp=true}}`